creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -347,129 +424,272 @@ class SnapbackSM {
         newReplicaSetSPIds.slice(1) // [secondary1, secondary2]
       )
 
-      // Sync data from existing primary to new secondary
-      phase = issueUpdateReplicaSetOpPhases.ENQUEUE_SYNCS
+      response.issuedReconfig = true
+
+      // Enqueue a sync for new primary to new secondaries. If there is no diff, then this is a no-op.
       await this.enqueueSync({
-        ...baseSyncRequestParams,
-        primaryEndpoint: primary,
-        secondaryEndpoint: newSecondary2
+        userWallet: wallet,
+        primaryEndpoint: newPrimary,
+        secondaryEndpoint: newSecondary1,
+        syncType: SyncType.Manual,
+        immediate: true
       })
+
+      await this.enqueueSync({
+        userWallet: wallet,
+        primaryEndpoint: newPrimary,
+        secondaryEndpoint: newSecondary2,
+        syncType: SyncType.Manual,
+        immediate: true
```

Comment:
why set this to manual + immediate? should be recurring + non-immediate right

Reply:
you're right -- it should be recurring. 

as for the immediate flag, my opinion is that we should probably prioritize syncs if a reconfig occurs cus we definitely know that there is missing data that needs to be replicated. I dont feel strongly

Reply:
issue with making this immediate is that it will be on same level as manual syncs issued by users during content upload. those need to always have priority and be processed before Reconfig syncs bc that is blocking a user's session
that's why i think we should not make these immediate